src/routers/genericRouters/routerFuzzy.ts

```diff
@@ -43,18 +43,21 @@ export class RouterFuzzy extends RouterFilterBy implements Router {
         LIMIT ${page * QUERY_LIMIT}, ${QUERY_LIMIT}
         RETURN { ${this.dbReturnStatements} }
     `
-
     const cursor = await db.query(query)
     return await cursor.all()
   }
 
-  async autocompleteSearch (term: string, page: number, apiReturn: boolean = false, customFilter: string = ''): Promise<any[]> {
+  async autocompleteSearch (term: string, page: number, apiReturn: boolean = false, customFilter: string = '', overWriteSearchField: string = ''): Promise<any[]> {
```

Comment:
Can you write the function documentation for function usage, parameters, returned value. It is hard for me to understand what is the meaning of some of the parameters. Thank you!

Reply:
added :)